fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -1148,8 +1149,7 @@ private void updateDBRunningTxnNum(TransactionStatus preStatus, TransactionState
                 && (curTxnState.getTransactionStatus() == TransactionStatus.PREPARE
                 || curTxnState.getTransactionStatus() == TransactionStatus.COMMITTED)) {
             ++txnNum;
-        } else if (preStatus != null
-                && (preStatus == TransactionStatus.PREPARE
+        } else if ((preStatus == TransactionStatus.PREPARE
```

Comment:
Why removing the `preStatus != null` check?

Reply:
because `preStatus == TransactionStatus.PREPARE` contains the `preStatus != null` check